KMB/kmb_massload.py

```diff
@@ -153,6 +135,25 @@ def parser(dom, A):
     return A
 
 
+def process_tags(entry, dom, label, xml_tag):
+    """
+    Process tags of a given type.
+
+    :param entry: the dict of parsed data for the image
+    :param dom: the dom being analysed
+    :param label: the label under which the processed tags should be stored
+    :param xml_tag: the xml tag name to search for
+    """
+    entry[label] = []
+    elements = dom.getElementsByTagName(xml_tag)
+    for element in elements:
+        try:
+            entry[label].append(element.childNodes[0].data.strip())
+        except IndexError:
+            # Means data for this field was mising
```

Comment:
It happens every now and then. It is due to (AFAIK) errors in the RAÄ data. My plan was that by outputting these we could possibly report them back to RAÄ.

I've filed [T165556](https://phabricator.wikimedia.org/T165556) to make sure this (and some other output you've mentioned) gets logged.